examples/cashiers-check/tests/cashiers-check.js

```diff
@@ -1,7 +1,9 @@
 const anchor = require("@project-serum/anchor");
 const serumCmn = require("@project-serum/common");
-const TokenInstructions = require("@project-serum/serum").TokenInstructions;
 const assert = require("assert");
+const { TOKEN_PROGRAM_ID, Token } = require("@solana/spl-token");
```

Comment:
`Token` can be removed right?

Reply:
true.   i just pushed a change to take it out.